src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="wrap">
+      <form
+        class="form big--form"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `big--form` seems to describe the size rather than the content or function.